lib/internal/cluster/RoundRobinHandle.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+const assert = require('assert');
+const net = require('net');
+const { sendHelper } = require('internal/cluster/utils');
```

Comment:
Lexical sort (in ASCII order/posix "C" locale)  on the statements (which ends up being by name of variable, but variable name usually maps 1:1 to the require name, es6 requires aside) (`:sort` in vim, pipe block through `!sort`, whatever your editor likes). No grouping, too complicated to document and reproduce.